applications/connecting_applications_to_services/getting-started-with-service-binding.adoc

```diff
@@ -17,7 +17,7 @@ The {servicebinding-title} manages the data plane for workloads and backing serv
 * You have access to an {product-title} cluster using an account with `cluster-admin` permissions.
 * You have installed the `oc` CLI.
 * You have installed {servicebinding-title} from OperatorHub.
-* You have installed the Crunchy Postgres for Kubernetes Operator from OperatorHub using the *v5* Update channel. The installed Operator is available in an appropriate namespace, such as the `my-petclinic` namespace.
+* You have installed the 5.1.2 version of the Crunchy Postgres for Kubernetes Operator from OperatorHub using the *v5* Update channel. The installed Operator is available in an appropriate namespace, such as the `my-petclinic` namespace.
```

Comment:
```suggestion
* You have installed the 5.1.2 version of the Crunchy Postgres for Kubernetes Operator from OperatorHub using the *v5* update channel. The installed Operator is available in an appropriate namespace, such as the `my-petclinic` namespace.
```

Reply:
This is how it appears in the UI. So retaining this content as is.